monitoring/micrometer-prometheus/src/test/java/io/quarkus/ts/micrometer/prometheus/UsingMicroProfilePingPongResourceIT.java

```diff
@@ -13,25 +13,40 @@
 @QuarkusScenario
 public class UsingMicroProfilePingPongResourceIT {
 
+    private static final String PING_PONG = "ping pong";
     private static final String PING_PONG_ENDPOINT = "/using-microprofile-pingpong";
-    private static final String COUNTER_FORMAT = "simple_mp_total{scope=\"application\",} %s.0";
+    private static final String COUNTER_FORMAT = "simple_counter_mp_total{scope=\"application\",} %s.0";
+    private static final String GAUGE_FORMAT = "simple_gauge_mp{scope=\"application\",} %s";
+    private static final long DEFAULT_GAUGE_VALUE = 100;
 
     @Test
     public void testShouldReturnCountOne() {
-        whenCallPingPong();
+        whenCallPingPong("/counter", PING_PONG);
         thenCounterIs(1);
     }
 
-    private void whenCallPingPong() {
+    @Test
+    public void testShouldReturnDefaultGauge() {
+        whenCallPingPong("/gauge", "" + DEFAULT_GAUGE_VALUE);
```

Comment:
""+long is a nice hack, but String.valueOf, or even making DEFAULT_GAUGE_VALUE a Long and use toString method will make this code clearer